lldb/test/API/lang/cpp/namespace/main.cpp

```diff
@@ -34,7 +34,7 @@ namespace A {
         int myfunc (int a);
         int myfunc2(int a)
         {
-             return a + 2;
+                return a + 2; //just changing tab not much
```

Comment:
In general, we don't add diffs that are unrelated to the problem being solved by the PR.
On a similar note, this comment seems to be trying to communicate with reviewers, not future readers of this test file. As such, this is the kind of comment that should be part of the commit message / PR description, and not part of the test itself